src/logic/Operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    try {
+      return (one.times(two).div(100)).toString();
+    } catch (err) {
+      return "Can't find modulo as can't divide by 0.";
+    }
+  }
```

Comment:
- [ ] The `%` button should work as a modulus, not as a percentage. If I calculate `17 %(mod) 5` the output should be `2` but currently, it shows `.85`. Kindly fix the issue.

- [ ] The arithmetic operators **`('+', '-', '/' etc.)`** of your calculator are working but it is not showing on the display. Whenever you click on any arithmetic operators, It should be displayed on the calculator display.